packages/jest-haste-map/src/index.js

```diff
@@ -325,6 +325,28 @@ class HasteMap extends EventEmitter {
           throw new Error(message);
         }
         this._console.warn(message);
```

Comment:
Oh, yeah, I think in the initial PR the way it was implemented is that we moved this error handling up the stack so that consumers can decide what to do it. We may want to do that because otherwise, in RN, we won't be able to show a redbox. Most people don't look at the cli output during development. However, this can of course be done later.